helix-core/src/test/java/org/apache/helix/TestHelper.java

```diff
@@ -219,56 +219,9 @@ private static Method getMethod(String name) {
     return null;
   }
 
-  public static boolean verifyEmptyCurStateAndExtView(String clusterName, String resourceName,
```

Comment:
moved to dedicated class: org/apache/helix/common/zkVerifiers/ExternalViewBalancedVerifier.java